chromehangs/index.html

```diff
@@ -9,9 +9,9 @@
     <script type="text/javascript" src="https://code.jquery.com/jquery.js"></script>
     <script type="text/javascript" src="https://code.jquery.com/ui/1.10.4/jquery-ui.js"></script>
     <script type="text/javascript" src="chromehangs.js"></script>
-    <script src="http://jquery-csv.googlecode.com/git/src/jquery.csv.js"></script>
+    <script src="//jquery-csv.googlecode.com/git/src/jquery.csv.js"></script>
```

Comment:
what is the src="//<site name>/.." format? it defaults to http or https based on how the top-level page was loaded? why not always use https?